app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Shows a specific user</h1>
\ No newline at end of file
+<div class = specific-user-container>
+    <div class = "user-info-part">
+        <img src = <%=@user.photo%> alt = "User photo" class = "specific-user-img">
+        <div class = "specific-user-info-container">
+            <h1 class = "user-name"><%=@user.name%></h1>
+            <h4 class = "user-post-num">Number of posts : <%=@user.post_counter%></h4>
+        </div>
+    </div>
+
+    <div class = "bio-container">
+        <h2 class= "bio-title">Bio</h2>
+        <p class = "bio-text"><%=@user.bio%></p>
+    </div>
+
+    <div class = "specific-user-posts">
+        <%@user.recent_posts.each do |post|%>
+            <div class = "user-post">
+                <h3 class ="user-post-title"><%=post.title%></h3>
```

Comment:
- [ ] Great work so far on the user show page. However, you still should implement the availability of clicking a post title from the user's recent posts to take the user to this post's show page. 🙏